src/bin/main.rs

```diff
@@ -44,18 +44,18 @@ fn main() {
 
     // evaluate
     // Uncomment and debug when we merge the two branches
-    // if args.evaluate {
-    //     match eval::eval(&program, &input) {
-    //         Ok(_) => {
-    //             println!("Evaluation successful");
-    //             info!("Evaluation successful"); // todo, same as above
-    //         }
-    //         Err(error) => {
-    //             println!("Evaluation Error: {error}");
-    //             warn!("Error with Evaluation: {}", error); // todo, same as above
-    //         }
-    //     }
-    // }
+    if args.evaluate {
+        match eval::evaluate(&program, &input) {
+            Ok(_) => {
+                println!("Evaluation successful");
```

Comment:
here, you should also print the verdict, like `program accepts the input`, `program rejects the input`.

Reply:
you need to use the argument of `Ok(_)` to make a decision